app/controllers/settings/base_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Settings::BaseController < ApplicationController
+  include Authorization
```

Comment:
How about including this in Settings::StaticSettingsController as need it here. Might not in all other inherited controller. What do you say? Maybe you added for other reasons. Do let me know :)